src/main/java/com/robothand/highqualitybot/Config.java

```diff
@@ -0,0 +1,47 @@
+package com.robothand.highqualitybot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Hashtable;
+import java.util.Scanner;
+
+/**
+ * Config.java
+ *
+ * Reads in properties from a specified configuration file, and provides access to the values.
+ */
+public class Config {
+    private final File configFile;
+
+    public String TOKEN;
+    public String PREFIX;
+
+
+    public Config(String path) throws FileNotFoundException {
+        configFile = new File(path);
+
+        Scanner s = new Scanner(configFile);
+
+        while (s.hasNextLine()) {
+            String[] split = s.nextLine().split("=");
```

Comment:
I think setting the Scanner's token to `=`and parsing that way may be more efficient--that leads to less String operations.